packages/ref-core/src/cmip_ref_core/pycmec/metric.py

```diff
@@ -314,7 +333,10 @@ def _fill(cls, mdict: dict[Any, Any], mdims: dict[Any, Any], level: int = 0) ->
         dim_name = mdims[MetricCV.JSON_STRUCTURE.value][level]
         for key in mdims[dim_name].keys():
             if key not in mdict:
-                mdict[key] = {}
+                if level == len(mdims[MetricCV.JSON_STRUCTURE.value]) - 1:
+                    mdict[key] = -999.0
```

Comment:
This should be hoisted up as a constant. I dislike these kinds of constants that exist in the same space as the values.

Reply:
If we allow the missing keywords in the deepest dimension, the if statements including the -999 statement can be removed.

Reply:
Not needed anymore since we don't need to support missing values